.github/workflows/test-reporter.yml

```
@@ -0,0 +1,18 @@
+﻿name: Test Reporter
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)